tests/contracts/testMarket.nim

```diff
@@ -1,30 +1,47 @@
 import std/options
+import std/importutils
 import pkg/chronos
+import pkg/ethers/erc20
 import codex/contracts
 import ../ethertest
 import ./examples
 import ./time
 import ./deployment
 
+privateAccess(OnChainMarket) # enable access to private fields
```

Comment:
🟠 It doesn't look like you need access to private fields of `OnChainMarket` in this test? Am I missing something?

Reply:
Good eye, Mark! `market.rewardRecipient` is used in `switchAccount`, which is not an exported field.